packages/react-native-editor/__device-tests__/gutenberg-editor-audio.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Internal dependencies
+ */
+import { blockNames } from './pages/editor-page';
+import testData from './helpers/test-data';
+import { isAndroid } from './helpers/utils';
+
+describe( 'Gutenberg Editor Audio Block tests', () => {
+	it( 'should be able to add an audio block', async () => {
+		await editorPage.addNewBlock( blockNames.audio );
+		const block = await editorPage.getFirstBlockVisible();
+		await expect( block ).toBeTruthy();
+	} );
+
+	it( 'should add an audio file to the block', async () => {
+		const block = await editorPage.getFirstBlockVisible();
+
+		block.click();
+
+		if ( isAndroid ) {
+			await editorPage.driver.sleep( 5000 );
+		} else {
+			await editorPage.driver.sleep( 1000 );
+		}
+
+		await editorPage.chooseMediaLibrary();
+
+		const html = await editorPage.getHtmlContent();
+		expect( testData.audioBlockPlaceholder.toLowerCase() ).toBe(
```

Comment:
I think I made a mistake converting these expectations lately. It should've been `expect(received).toBe(expected)` for the diff in case of an error to display correctly.
`expect( html... ).toBe(testData...)` in this case. Do you think it makes sense to fix other places in this PR as well?

Reply:
Makes sense! I resolved this for the File and Audio block since the changes being done in this PR were related to both blocks. I will create another PR for the block tests that need these assertions inverted.

Reply:
I created a PR for the rest here https://github.com/WordPress/gutenberg/pull/28770